integrationtests/disk_test.go

```diff
@@ -74,40 +74,41 @@ func TestDiskAPIGroup(t *testing.T) {
 
 		defer diskCleanup(t, vhdxPath, mountPath, testPluginPath)
 		diskNum := diskInit(t, vhdxPath, mountPath, testPluginPath)
+		diskNumAsString := strconv.FormatInt(diskNum, 10)
 
-		out, err := runPowershellCmd(fmt.Sprintf("Get-Disk -Number %s | Set-Disk -IsOffline $true", diskNum))
+		out, err := runPowershellCmd(fmt.Sprintf("Get-Disk -Number %d | Set-Disk -IsOffline $true", diskNum))
 		require.NoError(t, err, "failed setting disk offline, out=%v", out)
 
-		getReq := &v1beta3.GetAttachStateRequest{DiskID: diskNum}
+		getReq := &v1beta3.GetAttachStateRequest{DiskID: diskNumAsString}
```

Comment:
this change is temporal, once we work in the DiskAPI and it accepts numbers I'll remove this

Reply:
maybe create an issue to track this?

Reply:
this was fixed in #140